src/main/java/org/isf/lab/gui/elements/PatientComboBox.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2020 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isf.lab.gui.elements;
+
+import org.isf.generaldata.MessageBundle;
+import org.isf.lab.model.Laboratory;
+import org.isf.patient.model.Patient;
+
+import javax.swing.*;
```

Comment:
once again enumerate the classes and order the imports; thanks

Reply:
Is there any config file for project imports in intellij?

Reply:
If you load the `OpenHospital-code-style-configuration.xml` file in the `.ide-settings\idea` directory it sets the import order:
````
   <option name="IMPORT_LAYOUT_TABLE">
      <value>
        <package name="" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="java" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="javax" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="org" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="com" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="liquibase" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="" withSubpackages="true" static="false" />
      </value>
    </option>
````

Use `Settings -> Editor->Code Style->Java`
![image](https://user-images.githubusercontent.com/1105445/103488907-8a1a7400-4dde-11eb-91d9-ecba549e67d2.png)

Reply:
Thanks!